test/public/logs/detail.test.js

```diff
@@ -147,6 +147,30 @@ module.exports = () => {
         expect(redirectedUrl).to.equal(`${url}/?page=env-details&environmentId=${environmentId}`);
     });
 
+    it('allows navigating to an associated LHC fill', async () => {
+        const logId = 1;
+        const fillNumbers = [5, 6];
+
+        // Navigate to a log detail view
+        await goToPage(page, 'log-detail', { queryParameters: { id: logId } });
+        const showAllButton = await page.$('#toggleCollapse');
+        await showAllButton.click();
+        await page.waitForTimeout(1000);
```

Comment:
No, no more wait for timeout. Please look at the test I added yesterday and update yours